### PR TITLE
Bms mitogen 259

### PR DESCRIPTION
**Pull request by Nonsense331**
i don't want to merge but i want to release this as v2.5.9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -425,7 +425,11 @@ In order to dramatically speed up ansible, you can install Mitogen: https://gith
 
     pip install -g mitogen
 
-Subspace will automatically detect this and update your ansible.cfg file so it is blazing fast.
+Subspace will automatically detect this and update your ansible.cfg file so it is blazing fast.  Sometimes this can cause issues with older servers that have weird pythons, so if you have mitogen installed locally but dont wan't to use it, you can set an environment variable:
+
+    DISABLE_MITOGEN=1 subspace provision staging
+
+
```
Comment (samsonasu) [inline README.md:432]: 🤖 RUBY POLICE 🤖

**Comment by samsonasu** [comment]
🚢

File: README.md (hunk 1 above)
Comment (Nonsense331) [reply README.md:432]: you wrote that lol

**Comment by Nonsense331** [comment]
Released as v2.5.9!

**Pull request closed without merging**